### ISSUE TITLE
3.0 - Lodge still has maintenance

### ISSUE THREAD

**Issue by Stalker012345** [LoneGazebo/Community-Patch-DLL#9430]
_1. Mod Version (X.Y.Z). Current Version: 3.0.0_

3.0

_2. DirectX Version_

_3. List of Other Mods_

[Other Mods.zip](https://github.com/LoneGazebo/Community-Patch-DLL/files/10076675/Other.Mods.zip)

_4. Describe the Issue_

![image](https://user-images.githubusercontent.com/20593976/203578430-f31f49ed-6a28-4abf-a7bd-1fc8a2fe7032.png)
Lodge still generates maintenance, confirmed in the money screen as well so its not just textual.
______________________________________________________________________________________
_5. Save Game From 1 Turn Before (ALWAYS ATTACH THIS IF POSSIBLE)_

_6. Logs (ALWAYS ATTACH THESE IF POSSIBLE)_

_7. CvMiniDump.dmp File (ATTACH IF REPORTING A GAME CRASH)_

_8. Steps to reproduce the Issue (Optional)_

_9. Screenshots of the Issue (Optional)_

**Comment by axatin** [comment]
It was herbalist that lost the maintenance, not lodge

**Comment by Stalker012345** [comment]
> Intentional. It was the herbalist that lost the maintenance, not the lodge

You are right, I had gotten the buildings mixed up. The Herbalist is the building that was supposed to lose its maintenance but did not....it still has a -1 maintenance.
